app/locales/zh-tw/translations.coffee

```diff
@@ -7,38 +7,38 @@ I18nTranslationsZhTw =
     "reviewing": "正在審查"
     "reviewed": "已審查完畢"
     "submitted": "已提交"
-    "messages_title" : "信息"
+    "messages_title" : "訊息"
     "select": "選擇"
-    "QuotaExceededError": "Site may not work in Safari's <b>private mode.</b> Please try</br><ul><li><a href='https://itunes.apple.com/in/app/goodcityadmin.hk/id1013288708?mt=8' style='color: black!important; background-color: #dee4eb !important;'>Downloading the iOS App</a></li><li>Using regular (not private) mode in Safari</li><li>Using Chrome's private browsing mode</li></ul>"
+    "QuotaExceededError": "抱歉，此網站未能支援Safari <b>private mode.</b> 請嘗試</br><ul><li><a href='https://itunes.apple.com/in/app/goodcityadmin.hk/id1013288708?mt=8' style='color: black!important; background-color: #dee4eb !important;'>Downloading the iOS App</a></li><li>Using regular (not private) mode in Safari</li><li>Using Chrome's private browsing mode</li></ul>"
```

Comment:
Need Chinese translation for `Downloading the iOS App`, `Using regular (not private) mode in Safari` and `Using Chrome's private browsing mode`